docs/194-inicjalizacja.md

```diff
@@ -0,0 +1,87 @@
+# Inicjalizacja bota
+  
+## Punkt wejściowy
```

Comment:
tak jeszcze na koniec przypomne że w plikach .md znakiem końca linii jest podwójna spacja na końcu linii lub podwójny enter
my używamy podwójnej spacji na końcu każdej linii 
(nie dotyczy to bloków kodu)